### DISCUSSION TITLE
Migrator script from v8 to v9

### DISCUSSION THREAD

**Discussion by Linguisto** [directus/directus#3809] [Feature Requests]
Hi! I'm joining #3235 and I have an idea. Probably it worth creating a migrator script or something to easily migrate the system DB data, such as collection configurations, permissions and other incompatible stuff from v8 to v9. What do you think about it?

**Comment by MarcusMBX** [comment]
Take a look at https://github.com/directus/directus/discussions/3618
This patches the db to D9. Might be helpful.
I attempted to use a node.js script fetching data from D8 api and sending it to D9 api but patching the db seems a lot faster.

**Comment by benhaynes** [comment]
We have something in the works, although it may be released as "unofficially" supported. Could be out as soon as later this week... but needs a bit more testing.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
